ReactWindows/ReactNative/Views/TextInput/ReactPasswordBoxManager.cs

```diff
@@ -195,6 +196,17 @@ public void SetPlaceholder(PasswordBox view, string placeholder)
         }
 
         /// <summary>
```

Comment:
@rozele For the time being we don't have a way to invoke the yellow dialog from native code... Let me know how you'd like to proceed in the short-term....